app/services/attribute_asserter.rb

```diff
@@ -15,10 +15,11 @@ class AttributeAsserter
     :phone
   ].freeze
 
-  def initialize(user, service_provider, authn_request)
+  def initialize(user, service_provider, authn_request, decrypted_pii)
     self.user = user
     self.service_provider = service_provider
     self.authn_request = authn_request
+    @decrypted_pii = decrypted_pii
```

Comment:
I have used the leading underscore for memoized private instance variables that should not be accessed directly anywhere else in the code except for via the memoized method. That doesn't apply here, since we do want to use the instance var elsewhere. I can make it a private accessor if that would be clearer.